doc/centos-installation.md

```diff
@@ -0,0 +1,80 @@
+# APIcast v2 installation on CentOS
+
+This document explains how to install and run APIcast v2 on a clean CentOS.
+
+## Install OpenResty and dependencies
+
+OpenResty provides official pre-built packages for CentOS, the latest installation instructions are provided in [OpenResty documentation](https://openresty.org/en/linux-packages.html).
+
+You will need to create a file `/etc/yum.repos.d/OpenResty.repo` with the following content:
+
+```
+[openresty]
+name=Official OpenResty Repository
+baseurl=https://copr-be.cloud.fedoraproject.org/results/openresty/openresty/epel-$releasever-$basearch/
+skip_if_unavailable=True
+gpgcheck=1
+gpgkey=https://copr-be.cloud.fedoraproject.org/results/openresty/openresty/pubkey.gpg
+enabled=1
+enabled_metadata=1
+```
+
+Install OpenResty and the `resty` command-line utility, which is required for APIcast v2:
+
+```shell
+sudo yum install openresty openresty-resty
+```
+You can learn more about these and other OpenResty packages in [OpenResty documentation](https://openresty.org/en/rpm-packages.html).
+
+APIcast v2 uses LuaRocks for managing Lua dependencies. As it is not in the standard Yum repositories, you must first enable the [EPEL](https://fedoraproject.org/wiki/EPEL) (Extra Packages for Enterprise Linux) package repository with the following command:
+
+```shell
+sudo yum install epel-release
+```
+
+Install LuaRocks:
+```shell
+sudo yum install luarocks
+```
+
+If you are using OAuth authentication method, you will also need to install Redis. This can be done with `sudo yum install redis` command (note that you will need the [EPEL](https://fedoraproject.org/wiki/EPEL) for this).
+
+## Install and run APIcast v2
+
+To use the latest APIcast version, you can check out the `v2` branch of the [APIcast GitHub repository](https://github.com/3scale/apicast).
+
+If you want to use a specific version of APIcast, check the APIcast [releases page](https://github.com/3scale/apicast/releases). You can either download the source code from there, or checkout a specific tag with `git`, for example:
+
+```shell
+git checkout tags/v2.0.0-rc1
```

Comment:
I fee like mentioning git is really not needed. Github offers downloads for releases in tar.gz that are checkout from a repo.